web/vital_records/templatetags/form_helpers.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+@register.filter
+def label_with_required(field):
+    label = field.label or ""
+    required = ""
+    if field.field.required:
+        required = '<abbr title="required" class="text-danger">*</abbr>'
```

Comment:
This `abbr` style comes from USWDS and W3C reccs.

Reply:
In the future we can/should move this into the `web.core` app since it could be used on many different forms. Very nice :+1: